tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -41,7 +41,25 @@
 <template is="dom-if" if="[[_dataNotFound]]">
   <div style="max-width: 540px; margin: 80px auto 0 auto;">
     <h3>No profile data was found.</h3>
-    <p>The dashboard is internal only; coming soon!</p>
+    <p>To collect a profile, you need to run your model on Google Cloud TPUs and
+    capture the trace information while your model is running. You may want to
+    check out the
+    <a href="https://github.com/tensorflow/tensorboard/blob/1.6/tensorboard/plugins/profile/README.md">README</a>
+    and perhaps the
+    <a
+     href="https://cloud.google.com/tpu/docs/cloud-tpu-tools"
```

Comment:
Nit: HTML style is 100 column limit. say things like this:

```html
<doodle foo="bar"
        omg="lol>a</doodle>
```

Per go/htmlstyle even though we currently aren't compliant in many places.

Note: We sometimes need to be ugly because, unlike Soy, Polymer doesn't let us control whitespace.

Reply:
This is a cherry-pick of docs written by Qiumin in #957.  If you think these edits are release-blocking, please send a PR with changes to me or Qiumin ASAP and I will cherry-pick it into the release PR once it's merged into master - I'm not editing this in the release PR.  Ditto for all the other comments on this file.